tsdb/head.go

```diff
@@ -1906,13 +1907,9 @@ type sample struct {
 	v float64
 }
 
-func (s sample) T() int64 {
-	return s.t
-}
-
-func (s sample) V() float64 {
-	return s.v
-}
+func newSample(t int64, v float64) tsdbutil.Sample { return sample{t, v} }
```

Comment:
This doesn't appear to be used.

Reply:
It is used in `ExpandSamples` to make sure we compare correct `sample` types together (: 

![image](https://user-images.githubusercontent.com/6950331/87146872-867cb900-c2a3-11ea-923f-f9ebaa9f1727.png)